code/modules/arcade/prize_datums.dm

```diff
@@ -0,0 +1,343 @@
+
+var/global/datum/prizes/global_prizes = new
+
+/datum/prizes
+	var/list/datum/prize_item/prizes = list()
```

Comment:
Yes they can, I based this off the merch store computer's setup at cargo, and that uses typed lists. (https://github.com/ParadiseSS13/Paradise/blob/master/code/modules/store/store.dm#L22)